admin/assets/javascripts/global-scripts.js

```diff
@@ -30,7 +30,13 @@ $(function () {
   if ($('#pupilsList').length > 0) disableCheckAll('#pupilsList')
   if ($('#attendanceList').length > 0) disableCheckAll('#attendanceList')
 
-  $('input:file').on('change', function () {
+  $('.upn-text > a').on('click', function () {
```

Comment:
Please revert the style change associated with the changes to the UPN text on add/edit pupil. This is covered under a different ticket using `<detail><summary>` tags